bigfoot/views.py

```diff
@@ -19,8 +20,21 @@ def about(request):
 
 
 def sightings_partial(request):
-    return render(request, '_sightings.html', {
-        'foo': 'bar',
+    requested_page = 1
+    if request.GET['page'] is not None:
+        requested_page = int(request.GET['page'])
```

Comment:
If I am not mistaken this part can be written like:

```
requested_page = request.GET.get('page', 1)
```
which is a bit more Pythonic.